app/views/scoreboards/show.html.erb

```diff
@@ -73,19 +75,14 @@
 <% end %>
 
 
-<%= form_for @cud, url: course_course_user_datum_path(@course, @cud) do |f| %>
-<% if @cud.errors.any? %>
```

Comment:
Unnecessary since a user shouldn't be able to view the scoreboard if they have errors. `authorize_user_for_course` would have redirected users to the edit info page.